subsys/net/l2/ethernet/gptp/gptp_messages.c

```diff
@@ -178,7 +209,11 @@ struct net_pkt *gptp_prepare_sync(int port)
 	hdr->reserved2 = 0;
 
 	/* Ethernet configuration. */
+#if defined(CONFIG_NET_GPTP_VLAN)
```

Comment:
here it would be:
```
if (IS_ENABLED(CONFIG_NET_GPTP_VLAN)) {
        setup_eth_vlan_hdr(iface, pkt);
} else {
        eth->type = htons(NET_ETH_PTYPE_PTP);
}
```